tap_linkedin_ads/schemas/creatives.json

```diff
@@ -17,183 +29,99 @@
         "integer"
```

Comment:
I don't see this `campaign_id` in the given [sample response](https://learn.microsoft.com/en-us/linkedin/marketing/integrations/ads/account-structure/create-and-manage-creatives?view=li-lms-2023-01&tabs=http#sample-response-1). Is it being generated in the code ?

Reply:
Yes, it is generated in the code. It acts as a foreign key for the campaign table. Ref - [link](https://docs.microsoft.com/en-us/linkedin/marketing/integrations/ads/account-structure/create-and-manage-creatives#search-for-creatives). Same applies for `account_id`.